src/web/wms/src/main/java/org/geoserver/wms/web/data/StylePage.java

```diff
@@ -79,12 +79,7 @@ protected Component headerPanel() {
         header.add(removal = new SelectionRemovalLink("removeSelected", table, dialog) {
             @Override
             protected StringResourceModel canRemove(CatalogInfo object) {
-                StyleInfo s = (StyleInfo) object;
-                if ( StyleInfo.DEFAULT_POINT.equals( s.getName() ) || 
-                    StyleInfo.DEFAULT_LINE.equals( s.getName() ) || 
-                    StyleInfo.DEFAULT_POLYGON.equals( s.getName() ) || 
-                    StyleInfo.DEFAULT_RASTER.equals(s.getName()) ||
-                    StyleInfo.DEFAULT_GENERIC.equals(s.getName())) {
+                if (isDefaultStyle(object)) {
```

Comment:
No big deal, as said, the pull request is a valid improvement as is, just make clear in Jira what actually has been modified